app/models/crosswalk_issue.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class CrosswalkIssue < ApplicationRecord
+  belongs_to :crosswalk
+  belongs_to :ipeds_hd
+  belongs_to :weam
+
+  # rubocop:disable Metrics/MethodLength
+  def self.rebuild
+    sql = <<-SQL
+      INSERT INTO crosswalk_issues (
+        weam_id,
+        crosswalk_id,
+        ipeds_hd_id
+      )
+      SELECT
+        weams.id,
+        crosswalks.id,
+        ipeds_hds.id
+      FROM weams
+        LEFT OUTER JOIN ipeds_hds ON  weams.cross = ipeds_hds.cross
+          OR weams.ope = ipeds_hds.ope
+        LEFT OUTER JOIN crosswalks ON weams.facility_code = crosswalks.facility_code
+      WHERE
+        (institution_of_higher_learning_indicator = true OR non_college_degree_indicator = true)
+        AND NOT(
```

Comment:
Can you tell me what these `AND NOT` conditionals are doing in relation to the Acceptance Criteria?  

Are we eliminating valid records as opposed to selecting invalid records?

Reply:
We're looking for partial matches only, so we are excluding cases where:

- `weams` records fully matches across `ipeds_hds`, and `crosswalks`
- `weams` records without any partial matches across `ipeds_hds`, and `crosswalks`

If you're questioning the story AC I can ask to have it updated to reflect this.

Reply:
I was just trying to mentally map the AC to the SQL and having trouble.

Reply:
Yes, I completely understand how this might seem a little convoluted but we're trying to get the customer a view into this data so they can more easily review the "issues" as first step towards starting the discussion about how these might be resolved. Right now this is a manual process on their and and we're trying to be as "agile" as we can.

There's also a follow up PR that is waiting on this one which adds some additional cases.